src/lint/rules/spacing.rs

```diff
@@ -259,9 +272,12 @@ impl SpPunctRule {
                         punct_range.row_start, after_range.unwrap().row_end,
                         punct_range.col_start, after_range.unwrap().col_end,
                     );
-                    let dmlerror = LocalDMLError {
-                        range: error_range,
-                        description: Self::description().to_string(),
+                    let dmlerror = DMLStyleError {
```

Comment:
Doing this more than once, so could be worth implementing a fn constructor on DMLStyleError so you do not need to write out the LocalDMLError construct every time

Reply:
Thanks for catching this. We already have a function that takes care of this, but I need to replace all of these that remain. Will update it shortly.

Reply:
Done